community/usbutils/sources

```diff
@@ -0,0 +1 @@
+https://github.com/gregkh/usbutils/archive/v012.tar.gz
```

Comment:
https://www.kernel.org/pub/linux/utils/usb/usbutils/usbutils-012.tar.xz

This source should allow you to drop `autogen.sh`.

Reply:
I changed the source to this (sans the www.) but there was not a `configure` file in the source. Unless I'm missing something `autogen.sh` will still need to be run with this source.

Reply:
Understood. :+1: